src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlClientEventSource.cs

```diff
@@ -27,440 +29,616 @@ internal class SqlClientEventSource : EventSource
         /// Defines EventId for EndExecute (Reader, Scalar, NonQuery, XmlReader).
         /// </summary>
         private const int EndExecuteEventId = 2;
+
+        /// <summary>
```

Comment:
These comments really don't add any value. I know they're enforced by the build. Is there a way we can configure the rule so that only public methods are required to have doc comments?

Reply:
I agree with Wraith, since the class itself is internal, none of the keywords and their API documentation would show on IntelliSense. those could only be referenced by numbers only.

Reply:
It's not for public, it's for developers and maintainers.
Inline documentation adds a lot of value for teams where developers are not always the same!